cloud/src/meta-service/meta_service.cpp

```diff
@@ -1642,6 +1642,8 @@ void MetaServiceImpl::get_tablet_stats(::google::protobuf::RpcController* contro
 #ifdef NDEBUG
         // Force data size >= 0 to reduce the losses caused by bugs
         if (tablet_stats->data_size() < 0) tablet_stats->set_data_size(0);
+        if (tablet_stats->index_size() < 0) tablet_stats->set_index_size(0);
```

Comment:
what is the relationship between data_size, index_size and segment_size
do we need a separate segment_size?

Reply:
index_size + segment_size = data_size

Reply:
The data currently only shows the data size.

Reply:
By adding the index size and segment size, it becomes clear that the index size + segment size = data size. Without the index size, it's difficult to explain this clearly.